dump/dump_collect.cpp

```diff
@@ -95,7 +96,7 @@ void collectDumpFromSBE(struct pdbg_target* chip,
 {
     using namespace phosphor::logging;
     auto chipPos = pdbg_target_index(chip);
-    bool isOcmb = openpower::phal::sbe::is_ody_ocmb_chip(chip);
+    bool isOcmb = is_ody_ocmb_chip(chip);
```

Comment:
please limit changes only for the current scope

Reply:
ok